internal/commands/secrets/secrets_push.go

```diff
@@ -25,6 +26,16 @@ type SecretsPushOptions struct {
 	Force       bool
 }
 
+var secretsPushExample = templates.Examples(`
+	# Push secrets:
+
+    # This will push secrets for "squibby" service
+    ize secrets push squibby
+    
+    # This will push secrets for "squibby" service from a "example-service.json" file to the AWS SSM storage with force option(values will be overwritten if exist)
+	ize secrets push squibby --backend ssm --file string example-service.json --force
```

Comment:
`--file string` is an incorrect parameter (string is a type). Please ensure it's specified correctly here.